
DSM does not have the default mime-type files available,
need to bring our own for the module to work.

--- Lib/mimetypes.py.orig	2011-10-16 21:07:51.000000000 +0200
+++ Lib/mimetypes.py	2011-10-16 21:07:58.000000000 +0200
@@ -38,15 +38,7 @@
 ]

 knownfiles = [
-    "/etc/mime.types",
-    "/etc/httpd/mime.types",                    # Mac OS X
-    "/etc/httpd/conf/mime.types",               # Apache
-    "/etc/apache/mime.types",                   # Apache 1
-    "/etc/apache2/mime.types",                  # Apache 2
-    "/usr/local/etc/httpd/conf/mime.types",
-    "/usr/local/lib/netscape/mime.types",
-    "/usr/local/etc/httpd/conf/mime.types",     # Apache 1.2
-    "/usr/local/etc/mime.types",                # Apache 1.3
+    "@INSTALL_PREFIX@/etc/mime.types",
     ]

 inited = False
